hack/hypernetes/hyper.sh

```diff
@@ -29,7 +29,9 @@ group = "root"
 clear_emulator_capabilities = 0
 EOF
 
-	curl -sSL http://hypercontainer.io/install | bash
+	if curl -sSl http://hypercontainer.io/install | bash || curl -sSL http://hypercontainer.io/install | bash | grep "You are using the newest hyper" ; then
```

Comment:
Could we extract the check out here? e.g. check whether hyperctl is installed locally and check its version. If not latest, then install via `http://hypercontainer.io/install`.